blocks/library/freeform/freeform-block.js

```diff
@@ -200,6 +250,25 @@ export default class FreeformBlock extends wp.element.Component {
 		}
 	}
 
+	onScroll() {
+		if ( this.toolbarElem ) {
+			let blockControls = findDOMNode( this.toolbarElem );
+			while ( blockControls ) {
+				if ( blockControls.classList.contains( 'editor-visual-editor__block-controls' ) ) {
+					break;
+				}
+				blockControls = blockControls.parentElement;
+			}
```

Comment:
What element are we trying to get here? Could we use `element.closest()` instead of looping like this (We just need to include the polyfill `import 'element-closest';`?